flipperdevices/flipperzero-firmware
# [FL-2737] Dolphin level thresholds update

# What's new

- New level threshold values

# Verification 

- Enable level debug menu
- Change score counter and verify level is changing correctly

# Checklist (For Reviewer)
